apps/etterlatte-vedtaksvurdering/src/test/kotlin/vedtaksvurdering/VedtakBehandlingServiceTest.kt

```diff
@@ -1,4 +1,4 @@
-package no.nav.etterlatte.vedtaksvurdering
+package vedtaksvurdering
```

Comment:
Høyreklikk pakken i IDEA, run all tests tok _ikke_ med seg denne testklassen pga annen pakke. Kjenner ikke bakgrunnen for at det over omtrent hele kildekoden er mismatch i pakkenavn og fysisk lokasjon (og IDEA klager litt), men stemmer for at vi begynner gradvis å fikse det til -> `no.nav.etterlatte.<domene>.`

Reply:
Gikk for det samme som resten i pakken i denne omgang da.

Reply:
Ja, vi har en oppryddingsjobb med pakkenavnene.

Reply:
Prøvde å gå en runde på det der for lenge siden, men var ikke helt enighet rundt det av en eller annen grunn..

Reply:
Som utvikler forventer jeg at samtlige tester i treet under pakken/katalogen jeg velger i IDEA kjøres (og ikke bare i gradle), så det skal imho noe særs tungvektig grunn til for ikke å ha disse tingene i orden.